recipes/reentry/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "reentry" %}
+{% set version = "1.0.2"%}
+{% set sha256 = "d981bf90c46a11819ff35bea227e4337019ae3b7719038b51e0e21180321c656"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #fn: {{ name }}-{{ version }}.tar.gz
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+  git_url: https://github.com/DropD/reentry.git
+  git_rev: v1.0.2
+
+
+build:
+  number: 0
+  preserve_egg_dir: True
+  entry_points:
+    - reentry = reentry.cli:reentry
+
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools >=18.5
+
+  run:
+    - python
+    - setuptools >=18.5
```

Comment:
Is `setuptools` actually required to run the package? I didn't see any imports outside of `setup.py`, but I could have missed it.